src/core/preset/src/preset.module.ts

```diff
@@ -13,4 +13,30 @@ import { registerLibraryConfigReplacementExtension } from './preset-substitution
     },
   ],
 })
-export class FormlyPresetModule {}
+export class FormlyPresetModule {
+  static forTesting(presetNames: string[]): ModuleWithProviders<FormlyPresetModule> {
```

Comment:
instead of making it for testing purpose only, I'd prefer to provide a way to define `presets` here. For naming we could use `forRoot` or `withConfig`.